### ISSUE TITLE
[Suggestion] Config option for mob spawn weights

### ISSUE THREAD

**Issue by DWX12** [Daveyx0/PrimitiveMobs#243]
I have enjoyed this mod for quite sometime and was surprised in seeing that there is no spawn weight system nor other spawning options such as min and max group amounts. This is something I believe we would all like to see so we can limit spawning of specific mobs without having to disable them completely.

Thank you for your hard work and effort.

**Comment by Winddbourne** [comment]
Yes! This is needed. I would love to have flame spewers swimming about in the nether but right now with them on you can't walk three squares without seeing one . . . I'd like them to be a RARE mob.  

I'm going to try to use biome tweaker to remove them and then add them in with a low spawn weight but if that doesn't work I'll have to just turn them off . . . if it's a question of having them in such numbers it lags the server and prevents players from going near the lava seas or not having them at all I just have to ditch them in the interest of fun. :(

It's not a choice I want to make.
